eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/TranslationCollectorPass.php

```diff
@@ -21,19 +21,20 @@ class TranslationCollectorPass implements CompilerPassInterface
     public const LOCALES_MAP = [
         'de_DE' => 'de',
         'el_GR' => 'el',
+        'en_US' => 'en',
```

Comment:
It will use en_US if `ezplatform-i18n/ezplatform-i18n-en_US` is installed. 

Problem is the opposite situation (as in the case of [ezplatform-ee-demo](https://github.com/ezsystems/ezplatform-ee-demo/blob/master/composer.json#L59-L63) ): source translation is used only as a fallback, so it have the lowest priority no matter of `Accept-Language` value.

Reply:
> source translation is used only as a fallback

I understand that part :)
But why do you remap `en_US` to `en` ? We already have `en`, and it's actually `en_GB`.

Making sure that the `en` language (provided by default translation files) is registered in `available_translations` should be sufficient, you don't need this `'en_US' => 'en'` afaict.